docs/interest.md

```diff
@@ -21,41 +21,28 @@ use Pantheon\EI\WP\Interest;
 
 // Returns interest data from HeaderData class.
 $interest = Interest\get_interest();
-
-// Manually pass data to interest header.
-$data = ['HTTP_INTEREST' =>'Carl Sagan|Richard Feynman'];
-$interest = Interest\get_interest( $data );
-
-// Pass interest to header from query string, ei_interest
-$interest_from_query_string = isset( $_GET['ei_interest'] ) ? sanitize_text_field( $_GET['ei_interest'] ) : '';
-Interest\get_interest( [ 'HTTP_INTEREST' => $interest_query_string ] );
 ```
 
 ### `set_interest`
 
-Set the interest header key and data.
+Set the interest data in global header.
 
 #### Parameters
 
-`$key` _(array)_ Key for the header, or array of keys.
-
-`$data` _(array)_ Data to pass to the HeaderData class.
+`$data` _(array)_ Interest data to pass to the HeaderData class.
 
 #### Example
 
 ```php
 use Pantheon\EI\WP\Interest;
 
-$key = [ 'Comes' ];
-
-$data = [
-    'HTTP_IGNORED' => 'HTTP Ignored Entry',
-    'IGNORED_ENTRY' => 'Completely ignored entry',
-    'HTTP_SHOULD_BE_FOUND' => 'Should be found',
-    'HTTP_VARY' => 'Something, Wicked, This, Way',
-];
+// Manually pass data to interest header.
+$data = ['HTTP_INTEREST' =>'Carl Sagan|Richard Feynman'];
```

Comment:
Thought/Soapbox: As we get closer to launch, I'm thinking about the idea of including specific historic figures in our documentation and whether that's problematic. A (very) brief look at Sagan and Feynman doesn't throw up anything overtly problematic (Feynman worked on the Manhattan Project, which might be triggering, Sagan wrote about "testosterone poisoning" in men, which might be iffy) but what's perhaps more obvious is that they are both male scientists, which is already a field in which women have trouble gaining widespread recognition in the same way that their male counterparts do. 

We could probably avoid all that by just not putting names in here. There's a couple other places where the same name list is used, which should be changed, too, if we make this adjustment. It's one thing to use them for unit testing (arguably we shouldn't be using them there, either, in an open source project) but the documentation is much more public-facing.

Reply:
Good point! I did pull this from the [PHP library tests](https://github.com/pantheon-systems/pantheon-edge-integrations/blob/main/tests/HeaderDataTest.php#L126) where these names occur a few times. They're also in the [plugin tests](https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations/blob/main/tests/interestTest.php) as well. 

For these docs, I changed to a generic `Interest One|Interest Two` which should still demonstrate the functionality well enough, but let me know if further changes are needed.